js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetEdges.tsx

```diff
@@ -9,13 +9,20 @@ export const AssetEdges: React.FC<{
   highlighted: string | null;
   strokeWidth?: number;
   baseColor?: string;
-}> = ({edges, highlighted, strokeWidth = 4, baseColor = Colors.KeylineGray}) => {
+  viewportRect: {top: number; left: number; right: number; bottom: number};
+}> = ({edges, highlighted, strokeWidth = 4, baseColor = Colors.KeylineGray, viewportRect}) => {
   // Note: we render the highlighted edges twice, but it's so that the first item with
   // all the edges in it can remain memoized.
   return (
     <React.Fragment>
-      <AssetEdgeSet color={baseColor} edges={edges} strokeWidth={strokeWidth} />
       <AssetEdgeSet
+        color={baseColor}
+        edges={edges}
+        strokeWidth={strokeWidth}
+        viewportRect={viewportRect}
+      />
+      <AssetEdgeSet
+        viewportRect={viewportRect}
```

Comment:
It might be worth removing the `React.memo` from this component -- it's probably doing extra memoization for no reason now!